### ISSUE TITLE
Add a note-taking application to Kloügle

### ISSUE THREAD

**Issue by arugifa** [arugifa/klougle#3]
# User Story

As a user of Kloügle,
I would like to have access to a note-taking application,
So I can stop to use Google Keep.

# Acceptance Criteria

- The application's data should be saved on a persistent volume.

# Options

- [Standard Notes](https://standardnotes.org/)
- [Turtl](https://turtlapp.com/)

**Comment by arugifa** [comment]
I decided to go for [Standard Notes](https://standardnotes.org/), as this is a mature project, with a working web interface and Android/iOS applications. Also, the "Keep it simple" philosophy is in sync with the other projects already running in Kloügle.

An alternative would have been to use [Turtl](https://turtlapp.com/). But even if this project is a couple of years old now, it still looks not very stable (and there is no web interface...).

I already created a [Docker image](https://hub.docker.com/r/arugifa/standardnotes-server) for the Standard Notes server, as no official one is available. Next step is to make another image for the [dashboard](https://github.com/standardnotes/web).
